packages/cli/src/handlers/deploy.ts

```diff
@@ -31,19 +42,64 @@ export const deploy = async (options: DeployArgs): Promise<void> => {
     });
 };
 
+const createNewProject = async (
+    options: DeployHandlerOptions,
+): Promise<Project> => {
+    console.error('');
+    const absoluteProjectPath = path.resolve(options.projectDir);
+    const context = await getDbtContext({ projectDir: absoluteProjectPath });
+    const dbtName = friendlyName(context.projectName);
+
+    const answers = await inquirer.prompt([
+        {
+            type: 'input',
+            name: 'name',
+            message: `Add a project name or press enter to use the default: [${dbtName}] `,
+        },
+    ]);
+    const projectName = answers.name ? answers.name : dbtName;
+
+    console.error('');
+    const spinner = ora(`  Creating new project`).start();
+
+    try {
+        const project = await createProject({
+            ...options,
+            name: projectName,
+            type: ProjectType.NONE,
```

Comment:
Did we add this type in a different PR ? I can't see it in this PR

Reply:
I wonder if the none type should be under the DbtProjectType.

Reply:
Good catch, I pushed the change now. 

I think it makes sense to be on ProjectType, like `PREVIEW` , but maybe we should call it CLI rather than NONE, what do you think ? 

![image](https://user-images.githubusercontent.com/1983672/190093332-69a12439-8e69-4295-8b21-7e1eb20f2ded.png)